src/Controller/CommandLineController.ts

```diff
@@ -0,0 +1,96 @@
+import yargs from 'yargs'
+
+export interface ArgvData {
```

Comment:
Can you write documentation for this interface (even if its brief)? I had to read the rest of the file to figure out what it's used for.

Reply:
Fixed.